packages/flutter_adaptive_scaffold/test/adaptive_scaffold_test.dart

```diff
@@ -29,10 +30,11 @@ void main() {
     expect(primaryNav, findsNothing);
 
     expect(tester.getTopLeft(smallBody), Offset.zero);
-    expect(tester.getTopLeft(smallSBody), const Offset(150, 0));
+    expect(tester.getTopLeft(smallSBody), const Offset(200, 0));
```

Comment:
Since we use 
```dart
await tester.binding.setSurfaceSize(SimulatedLayout.mobile.size); // 400
```
instead of 
```dart
await tester.pumpWidget(await scaffold(width: 300, tester: tester));
```
the expected starting point of `smallSBody` is now 50 pixels to the right. Adding `100dp` adds `50dp` do both `smallBody` and `smallSBody` since they split the added space. Since `smallSBody` is displayed to the right of `smallBody` it will be shfited by `50dp`